site/content/en/docs/reference/labels-and-annotations.md

```diff
@@ -180,3 +180,14 @@ Example: `kueue.x-k8s.io/role-hash: "b54683bb"`
 Used on: [Plain Pods](/docs/tasks/run/plain_pods/).
 
 The annotation key is used as the name for a Workload podSet.
+
+
+### kueue.x-k8s.io/max-exec-time-seconds
```

Comment:
We sorted this list in alphabetical order. Could you please move it after `kueue.x-k8s.io/managed`?

Reply:
done